src/components/pages/Categories/Edit.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <app-category-edit
+    :category-name="categoryName"
+    :loading="loading"
+    :done-message="doneMessage"
+    :access="access"
+    :error-message="errorMessage"
+    @edit-value="editValue"
+    @handle-submit="handleSubmit"
+  />
+</template>
+
+<script>
+import CategoryEdit from '@Components/molecules/CategoryEdit/index.vue';
+
+export default {
+  components: {
+    appCategoryEdit: CategoryEdit,
+  },
+  computed: {
+    categoryId() {
+      let { id } = this.$route.params;
+      id = parseInt(id, 10);
+      return id;
```

Comment:
[imo]
letはなるべく使いたくないのでそのままreturnしてしまって良いかなと思いました！

```suggestion
      return parseInt(this.$route.params.id, 10);
```

Reply:
承知しました!